.github/workflows/maven.yml

```diff
@@ -0,0 +1,34 @@
+# This workflow will build a Java project with Maven, and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-java-with-maven
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: Java CI with Maven
+
+on:
+  push:
+    branches: [ "main" ]
```

Comment:
nit: technically this isn't necessary, because we won't push anything directly to `main`

Reply:
Solved in [https://github.com/BME-MIT-IET/iet-hf-2024-asd/pull/7/commits/dcf9749a7a87f6c764359b8202358802152a9085](url) commit.